cloudfoundry-operations/src/main/java/org/cloudfoundry/operations/services/Services.java

```diff
@@ -64,6 +64,13 @@
     Flux<ServiceInstance> listInstances();
 
     /**
+     * LList available offerings in the marketplace
+     *
+     * @return the service instances
+     */
+    Flux<ServiceOffering> listOfferings(GetMarketplaceRequest request);
```

Comment:
@antechrestos I'd like to unify this signature and types to be consistently "marketplace".  I think `ServiceOffering` is fine though.  I'll update.